CITATION.cff

```diff
@@ -1,43 +1,87 @@
-- xarray
-- metadata
+# This CITATION.cff file was generated with cffinit.
+# Visit https://bit.ly/cffinit to generate yours today!
 
+cff-version: 1.2.0
+title: cf_xarray
+message: >-
+  If you use this software, please cite it using these
+  metadata.
+type: software
 authors:
-- affiliation: National Center for Atmospheric Research, USA
-  family-names: Cherian
-  given-names: Deepak
-  orcid: https://orcid.org/0000-0002-6861-8734
-- affiliation: National Oceanography Centre, Southampton, UK
-  family-names: Almansi
-  given-names: Mattia
-  orcid: https://orcid.org/0000-0001-6849-3647
-- affiliation: Ouranos, Inc.
-  family-names: Bourgault
-  given-names: Pascal
-  orcid: https://orcid.org/0000-0003-1192-0403
-- affiliation: National Center for Atmospheric Research, USA
-  family-names: Kent
-  given-names: Julia
-  orcid: https://orcid.org/0000-0002-5611-8986
-- family-names: Magin
-  given-names: Justus
-- family-names: Thielen
-  given-names: Jon
-  orcid: https://orcid.org/0000-0002-5479-0189
-  affiliation: Iowa State University, Ames, IA, USA
-- affiliation: Axiom Data Science
-  family-names: Thyng
-  given-names: Kristen
-  orcid: https://orcid.org/0000-0002-8746-614X
-...
+  - affiliation: 'National Center for Atmospheric Research, USA'
+    family-names: Cherian
+    given-names: Deepak
+    orcid: 'https://orcid.org/0000-0002-6861-8734'
+  - affiliation: 'National Oceanography Centre, Southampton, UK'
+    family-names: Almansi
+    given-names: Mattia
+    orcid: 'https://orcid.org/0000-0001-6849-3647'
+  - affiliation: 'Ouranos, Inc.'
+    family-names: Bourgault
+    given-names: Pascal
+    orcid: 'https://orcid.org/0000-0003-1192-0403'
+  - affiliation: Axiom Data Science
+    family-names: Thyng
+    given-names: Kristen
+    orcid: 'https://orcid.org/0000-0002-8746-614X'
+  - family-names: Thielen
+    given-names: Jonathan
+    orcid: 'https://orcid.org/0000-0002-5479-0189'
+    affiliation: 'Iowa State University, Ames, IA, USA'
+  - family-names: Magin
+    given-names: Justus
+  - affiliation: 'National Center for Atmospheric Research, USA'
+    family-names: Kent
+    given-names: Julia
+    orcid: 'https://orcid.org/0000-0002-5611-8986'
+  - family-names: Fernandes
+    given-names: Filipe
+  - family-names: Caneill
+    given-names: Romain
+    orcid: 'https://orcid.org/0000-0001-6649-4275'
+    affiliation: Gothenburg University
+  - family-names: Mankoff
+    given-names: Ken
+    orcid: 'https://orcid.org/0000-0001-5453-2019'
+    affiliation: Goddard Institute for Space Studies
+  - family-names: Buntemeyer
+    given-names: Lars
+    orcid: 'https://orcid.org/0000-0002-0849-2404'
+    affiliation: Helmholtz-Zentrum Hereon
+  - family-names: Aoun
+    given-names: Abel
+    orcid: 'https://orcid.org/0000-0003-2289-2890'
+    affiliation: CERFACS
+  - family-names: Davis
+    given-names: Luke
+    orcid: 'https://orcid.org/0000-0003-1072-4638'
+    affiliation: Colorado State University
+  - family-names: Müller
```

Comment:
@MuellerSeb do you have an ORCID you wish to add?

Reply:
Thanks for reaching out to me!
```
orcid: 'https://orcid.org/0000-0001-9060-4008'
affiliation: Helmholtz Centre for Environmental Research - UFZ
```